content/lessons/genesis-pt2.mdx

```diff
@@ -5,8 +5,8 @@ contentType: 'challenge'
 
 # Let's make sense of this
 
-Use the following command to decode Satoshi's secret HEX message into ASCII, a more human readable format:
+To decode Satoshi's secret hidden in the message you saw in the last exercise, use the following command and replace \<scriptSigHex\> with the message to interpret it.
```

Comment:
I like how you're trying to remove some of the technical terms, but it's not consistent right now with the previous screen. The term `message` only appears here, the previous screen does not refer to any messages. Maybe review both screens together (as well as the next challenge which is also about decoding messages) and see what works across them all?

`ScriptSig HEX Representation` really is a mouthful, I hope we can find something simpler.

Reply:
Appreciate the effort here to make things more clear to the user :) Regarding discussions that involve copy, what does everyone think about moving those to Figma? We're lucky for now that we have a designer that reviews PRs, but that may not always be the case.

I don't have an issue with the original copy, but the proposed change is a bit long. Is there a way we can break it up into two, maybe three smaller sentences? I also agree with Christoph on the use of the term 'message'.

Something like: "Remember the message from the last exercise? Let's decode the secret that Satoshi left in it. Use the following command and replace \<scriptSigHex\> with the message. We need to do this because that message is currently in HEX format. This command will decode it to ASCII, something that is much easier for humans to read."

Another way we can mitigate some confusion is also through creative use of tool tips, which is more than just a simple copy change. This is another reason why I think Figma might be a better place for these kinds of discussions.

Reply:
Do we need to propose a process for submitting copy edits? I also think it's better to keep those discussion out of GitHub.

Reply:
yeah! let's put it on the agenda for tomorrow's call